src/androidTest/java/at/bitfire/vcard4android/contactrow/EventBuilderTest.kt

```diff
@@ -29,37 +30,37 @@ class EventBuilderTest {
     @Test
     fun testStartDate_FullDate() {
         EventBuilder(Uri.EMPTY, null, Contact().apply {
-            anniversary = Anniversary(Calendar.getInstance().apply {
-                set(1984, /* zero-based */ 7, 20)
-            })
+            anniversary = Anniversary(
+                LocalDate.of(1984, /* zero-based */ 8, 20)
```

Comment:
8 for August is not zero-based, the comment should be removed in all such lines

7 for August would be zero-based (0 = January), and because zero-based months are quite uncommon, I have added the comment in the original code